### PR TITLE
MSI - Adding ACM Observability tests

### PR DESCRIPTION
**Pull request by omrirh**

### CODE REVIEW

**Comment by omrirh** [comment]
/pj-rehearse pull-ci-CSPI-QE-MSI-acm-observability-v4.14-stage-multi-cluster-observability-4-14-rosa-acm-gcp-mngd-rosa-mngd

**Comment by omrirh** [comment]
/pj-rehearse pull-ci-CSPI-QE-MSI-acm-observability-v4.14-stage-multi-cluster-observability-4-14-rosa-acm-gcp-mngd-rosa-mngd

**Comment by omrirh** [comment]
/pj-rehearse pull-ci-CSPI-QE-MSI-acm-observability-v4.14-stage-multi-cluster-observability-4-14-rosa-acm-gcp-mngd-rosa-mngd

**Comment by omrirh** [comment]
/pj-rehearse pull-ci-CSPI-QE-MSI-acm-observability-v4.14-stage-multi-cluster-observability-4-14-rosa-acm-gcp-mngd-rosa-mngd

**Comment by omrirh** [comment]
/pj-rehearse pull-ci-CSPI-QE-MSI-acm-observability-v4.14-stage-multi-cluster-observability-4-14-rosa-acm-gcp-mngd-rosa-mngd

**Comment by omrirh** [comment]
/pj-rehearse pull-ci-CSPI-QE-MSI-acm-observability-v4.14-stage-multi-cluster-observability-4-14-rosa-acm-gcp-mngd-rosa-mngd

**Comment by omrirh** [comment]
/pj-rehearse pull-ci-CSPI-QE-MSI-acm-observability-v4.14-stage-multi-cluster-observability-4-14-rosa-acm-gcp-mngd-rosa-mngd

**Comment by omrirh** [comment]
/pj-rehearse pull-ci-CSPI-QE-MSI-acm-observability-v4.14-stage-multi-cluster-observability-4-14-rosa-acm-gcp-mngd-rosa-mngd

File: ci-operator/step-registry/cluster/install/cluster-install-commands.sh [hunk 1]
```diff
@@ -89,6 +89,10 @@ if [ $NUM_CLUSTERS -eq 1 ]; then
   grep 'cluster-id' "$CLUSTER_DATA_DIR/cluster_data.yaml" |  awk -F': ' '{print $2}' > "${SHARED_DIR}/cluster-id"
 fi
 
+if [ -n "$HUB_CLUSTER_DATA_DIR" ]; then
+  grep 'kubeconfig-path' "$HUB_CLUSTER_DATA_DIR/cluster_data.yaml" |  awk -F': ' '{print $2}' > "${SHARED_DIR}/hub-kubeconfig-path"
+fi
+
```
Comment (rnetser) [inline ci-operator/step-registry/cluster/install/cluster-install-commands.sh:95]: should not be here but in the tests part

File: ci-operator/step-registry/acm-observability/tests/acm-observability-tests-ref.yaml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+      cpu: 100m
+      memory: 200Mi
+  env:
+    - name: HUB_CLUSTER_NAME
+      default: "ci-acm-hub"
+      documentation: Name of the hub cluster to run ACM Observability tests against
+    - name: TEST_MARKER
+      default: "acm_observability"
+      documentation: Test marker for running ACM Observability testsuite
+  documentation: >-
```
Comment (rnetser) [inline ci-operator/step-registry/acm-observability/tests/acm-observability-tests-ref.yaml:16]: please add that the cluster should be installed using `cluster-install` step registry

**Review by rnetser — CHANGES_REQUESTED** [review]

File: ci-operator/step-registry/acm-observability/tests/acm-observability-tests-ref.yaml (hunk 2 above)
Comment (omrirh) [reply ci-operator/step-registry/acm-observability/tests/acm-observability-tests-ref.yaml:22]: done

**Comment by omrirh** [comment]
/pj-rehearse pull-ci-CSPI-QE-MSI-acm-observability-v4.14-stage-multi-cluster-observability-4-14-rosa-acm-gcp-mngd-rosa-mngd

**Comment by omrirh** [comment]
/pj-rehearse pull-ci-CSPI-QE-MSI-acm-observability-v4.14-stage-multi-cluster-observability-4-14-rosa-acm-gcp-mngd-rosa-mngd

**Comment by omrirh** [comment]
/pj-rehearse pull-ci-CSPI-QE-MSI-acm-observability-v4.14-stage-multi-cluster-observability-4-14-rosa-acm-gcp-mngd-rosa-mngd

**Comment by omrirh** [comment]
/pj-rehearse pull-ci-CSPI-QE-MSI-acm-observability-v4.14-stage-multi-cluster-observability-4-14-rosa-acm-gcp-mngd-rosa-mngd

**Pull request merged**